### ISSUE TITLE
Getting f5-networks:f5-big-ip-best:f5-bigip-virtual-edition-200m-best-hourly:14.1.206000

### ISSUE THREAD

**Issue by scshitole** [f5devcentral/terraform-azure-bigip-module#28]
I tried simple git clone and ran the terraform init, plan, apply

```
SEA-ML-00060393:bigip_azure_1nic_deploy shitole$ terraform init
Initializing modules...
- bigip in ../..
Downloading Azure/network-security-group/azurerm 3.5.0 for mgmt-network-security-group...
- mgmt-network-security-group in .terraform/modules/mgmt-network-security-group
Downloading Azure/vnet/azurerm 2.4.0 for network...
- network in .terraform/modules/network

Initializing the backend...

Initializing provider plugins...
- Finding hashicorp/azurerm versions matching "~> 2.0, > 2.28.0"...
- Finding hashicorp/random versions matching "> 2.3.0"...
- Finding hashicorp/null versions matching "> 2.1.2"...
- Finding hashicorp/template versions matching "> 2.1.2"...
- Installing hashicorp/azurerm v2.53.0...
- Installed hashicorp/azurerm v2.53.0 (signed by HashiCorp)
- Installing hashicorp/random v3.1.0...
- Installed hashicorp/random v3.1.0 (signed by HashiCorp)
- Installing hashicorp/null v3.1.0...
- Installed hashicorp/null v3.1.0 (signed by HashiCorp)
- Installing hashicorp/template v2.2.0...
- Installed hashicorp/template v2.2.0 (signed by HashiCorp)

Terraform has created a lock file .terraform.lock.hcl to record the provider
selections it made above. Include this file in your version control repository
so that Terraform can guarantee to make the same selections by default when
you run "terraform init" in the future.


Warning: Version constraints inside provider configuration blocks are deprecated

  on main.tf line 2, in provider "azurerm":
   2:   version = "~>2.0"

Terraform 0.13 and earlier allowed provider version constraints inside the
provider configuration block, but that is now deprecated and will be removed
in a future version of Terraform. To silence this warning, move the provider
version constraint into the required_providers block.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
SEA-ML-00060393:bigip_azure_1nic_deploy shitole$ terraform plan

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create
 <= read (data resources)

Terraform will perform the following actions:

  # data.azurerm_subnet.mgmt will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_subnet" "mgmt"  {
      + address_prefix                                 = (known after apply)
      + address_prefixes                               = (known after apply)
      + enforce_private_link_endpoint_network_policies = (known after apply)
      + enforce_private_link_service_network_policies  = (known after apply)
      + id                                             = (known after apply)
      + name                                           = "mgmt-subnet"
      + network_security_group_id                      = (known after apply)
      + resource_group_name                            = (known after apply)
      + route_table_id                                 = (known after apply)
      + service_endpoints                              = (known after apply)
      + virtual_network_name                           = (known after apply)

      + timeouts {
          + read = (known after apply)
        }
    }

  # azurerm_network_security_rule.mgmt_allow_http will be created
  + resource "azurerm_network_security_rule" "mgmt_allow_http" {
      + access                      = "Allow"
      + destination_address_prefix  = "*"
      + destination_port_range      = "80"
      + direction                   = "Inbound"
      + id                          = (known after apply)
      + name                        = "Allow_Http"
      + network_security_group_name = (known after apply)
      + priority                    = 201
      + protocol                    = "Tcp"
      + resource_group_name         = (known after apply)
      + source_address_prefixes     = [
          + "0.0.0.0/0",
        ]
      + source_port_range           = "*"
    }

  # azurerm_network_security_rule.mgmt_allow_https will be created
  + resource "azurerm_network_security_rule" "mgmt_allow_https" {
      + access                      = "Allow"
      + destination_address_prefix  = "*"
      + destination_port_range      = "8443"
      + direction                   = "Inbound"
      + id                          = (known after apply)
      + name                        = "Allow_Https"
      + network_security_group_name = (known after apply)
      + priority                    = 200
      + protocol                    = "Tcp"
      + resource_group_name         = (known after apply)
      + source_address_prefixes     = [
          + "0.0.0.0/0",
        ]
      + source_port_range           = "*"
    }

  # azurerm_network_security_rule.mgmt_allow_ssh will be created
  + resource "azurerm_network_security_rule" "mgmt_allow_ssh" {
      + access                      = "Allow"
      + destination_address_prefix  = "*"
      + destination_port_range      = "22"
      + direction                   = "Inbound"
      + id                          = (known after apply)
      + name                        = "Allow_ssh"
      + network_security_group_name = (known after apply)
      + priority                    = 202
      + protocol                    = "Tcp"
      + resource_group_name         = (known after apply)
      + source_address_prefixes     = [
          + "0.0.0.0/0",
        ]
      + source_port_range           = "*"
    }

  # azurerm_resource_group.rg will be created
  + resource "azurerm_resource_group" "rg" {
      + id       = (known after apply)
      + location = "eastus"
      + name     = (known after apply)
    }

  # null_resource.clusterDO[0] will be created
  + resource "null_resource" "clusterDO" {
      + id = (known after apply)
    }

  # random_id.id will be created
  + resource "random_id" "id" {
      + b64_std     = (known after apply)
      + b64_url     = (known after apply)
      + byte_length = 2
      + dec         = (known after apply)
      + hex         = (known after apply)
      + id          = (known after apply)
    }

  # module.bigip[0].data.azurerm_public_ip.f5vm01mgmtpip will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_public_ip" "f5vm01mgmtpip"  {
      + allocation_method       = (known after apply)
      + domain_name_label       = (known after apply)
      + fqdn                    = (known after apply)
      + id                      = (known after apply)
      + idle_timeout_in_minutes = (known after apply)
      + ip_address              = (known after apply)
      + ip_version              = (known after apply)
      + location                = (known after apply)
      + name                    = (known after apply)
      + resource_group_name     = (known after apply)
      + reverse_fqdn            = (known after apply)
      + sku                     = (known after apply)
      + zones                   = (known after apply)

      + timeouts {
          + read = (known after apply)
        }
    }

  # module.bigip[0].data.azurerm_resource_group.bigiprg will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_resource_group" "bigiprg"  {
      + id       = (known after apply)
      + location = (known after apply)
      + name     = (known after apply)
      + tags     = (known after apply)

      + timeouts {
          + read = (known after apply)
        }
    }

  # module.bigip[0].data.template_file.clustermemberDO1[0] will be read during apply
  # (config refers to values not yet known)
 <= data "template_file" "clustermemberDO1"  {
      + id       = (known after apply)
      + rendered = (known after apply)
      + template = <<-EOT
            {
                "schemaVersion": "1.0.0",
                "class": "Device",
                "async": true,
                "label": "Onboard BIG-IP",
                "Common": {
                    "class": "Tenant",
                    "mySystem": {
                        "class": "System",
                        "hostname": "${hostname}"
                    },
                    "myDns": {
                        "class": "DNS",
                        "nameServers": [
                                ${name_servers}
                        ],
                        "search": [
                            "f5.com"
                        ]
                    },
                    "myNtp": {
                        "class": "NTP",
                        "servers": [
                                ${ntp_servers}
                        ],
                        "timezone": "UTC"
                    }
                }
            }
        EOT
      + vars     = {
          + "hostname"      = (known after apply)
          + "name_servers"  = "\"169.254.169.253\""
          + "ntp_servers"   = "\"169.254.169.123\""
          + "search_domain" = "f5.com"
        }
    }

  # module.bigip[0].data.template_file.init_file[0] will be read during apply
  # (config refers to values not yet known)
 <= data "template_file" "init_file"  {
      + id       = (known after apply)
      + rendered = (known after apply)
      + template = <<-EOT
            
            #!/bin/bash -x
            
            # NOTE: Startup Script is run once / initialization only (Cloud-Init behavior vs. typical re-entrant for Azure Custom Script Extension )
            # For 15.1+ and above, Cloud-Init will run the script directly and can remove Azure Custom Script Extension 
            
            
            mkdir -p  /var/log/cloud /config/cloud /var/config/rest/downloads
            
            
            LOG_FILE=/var/log/cloud/startup-script.log
            [[ ! -f $LOG_FILE ]] && touch $LOG_FILE || { echo "Run Only Once. Exiting"; exit; }
            npipe=/tmp/$$.tmp
            trap "rm -f $npipe" EXIT
            mknod $npipe p
            tee <$npipe -a $LOG_FILE /dev/ttyS0 &
            exec 1>&-
            exec 1>$npipe
            exec 2>&1
            
            mkdir -p /config/cloud
              
            curl -o /config/cloud/do_w_admin.json -s --fail --retry 60 -m 10 -L https://raw.githubusercontent.com/f5devcentral/terraform-azure-bigip-module/dev_saketha_runtimeinit/config/onboard_do.json
            
            
            ### write_files:
            # Download or Render BIG-IP Runtime Init Config 
            
            cat << 'EOF' > /config/cloud/runtime-init-conf.yaml
            ---
            runtime_parameters:
              - name: USER_NAME
                type: static
                value: ${bigip_username}
            EOF
            
            if ${az_key_vault_authentication}
            then
               cat << 'EOF' >> /config/cloud/runtime-init-conf.yaml
              - name: ADMIN_PASS
                type: secret
                secretProvider:
                  environment: azure
                  type: KeyVault
                  vaultUrl: ${vault_url}
                  secretId: ${secret_id}
            EOF
            
            else
            
               cat << 'EOF' >> /config/cloud/runtime-init-conf.yaml
              - name: ADMIN_PASS
                type: static
                value: ${bigip_password}
            EOF
            fi
            
            cat << 'EOF' >> /config/cloud/runtime-init-conf.yaml
            pre_onboard_enabled:
              - name: provision_rest
                type: inline
                commands:
                  - /usr/bin/setdb provision.extramb 500
                  - /usr/bin/setdb restjavad.useextramb true
                  - /usr/bin/setdb setup.run false
            extension_packages:
              install_operations:
                - extensionType: do
                  extensionVersion: ${DO_VER}
                  extensionUrl: ${DO_URL}
                - extensionType: as3
                  extensionVersion: ${AS3_VER}
                  extensionUrl: ${AS3_URL}
                - extensionType: ts
                  extensionVersion: ${TS_VER}
                  extensionUrl: ${TS_URL}
                - extensionType: cf
                  extensionVersion: ${CFE_VER}
                  extensionUrl: ${CFE_URL}
            extension_services:
              service_operations:
                - extensionType: do
                  type: url
                  value: file:///config/cloud/do_w_admin.json
            post_onboard_enabled: []
            EOF
            # # Download
            #PACKAGE_URL='https://cdn.f5.com/product/cloudsolutions/f5-bigip-runtime-init/v1.1.0/dist/f5-bigip-runtime-init-1.1.0-1.gz.run'
            #PACKAGE_URL='https://cdn.f5.com/product/cloudsolutions/f5-bigip-runtime-init/v1.2.0/dist/f5-bigip-runtime-init-1.2.0-1.gz.run'
            for i in {1..30}; do
                curl -fv --retry 1 --connect-timeout 5 -L ${INIT_URL} -o "/var/config/rest/downloads/f5-bigip-runtime-init-1.1.0-1.gz.run" && break || sleep 10
            done
            # Install
            bash /var/config/rest/downloads/f5-bigip-runtime-init-1.1.0-1.gz.run -- '--cloud azure'
            # Run
            f5-bigip-runtime-init --config-file /config/cloud/runtime-init-conf.yaml
        EOT
      + vars     = {
          + "AS3_URL"                     = "https://github.com/F5Networks/f5-appsvcs-extension/releases/download/v3.25.0/f5-appsvcs-3.25.0-3.noarch.rpm"
          + "AS3_VER"                     = "v3.25.0"
          + "CFE_URL"                     = "https://github.com/F5Networks/f5-cloud-failover-extension/releases/download/v1.7.1/f5-cloud-failover-1.7.1-1.noarch.rpm"
          + "CFE_VER"                     = "v1.7.1"
          + "DO_URL"                      = "https://github.com/F5Networks/f5-declarative-onboarding/releases/download/v1.18.0/f5-declarative-onboarding-1.18.0-4.noarch.rpm"
          + "DO_VER"                      = "v1.18.0"
          + "INIT_URL"                    = "https://cdn.f5.com/product/cloudsolutions/f5-bigip-runtime-init/v1.2.0/dist/f5-bigip-runtime-init-1.2.0-1.gz.run"
          + "TS_URL"                      = "https://github.com/F5Networks/f5-telemetry-streaming/releases/download/v1.17.0/f5-telemetry-1.17.0-4.noarch.rpm"
          + "TS_VER"                      = "v1.17.0"
          + "az_key_vault_authentication" = "false"
          + "bigip_password"              = (known after apply)
          + "bigip_username"              = "bigipuser"
          + "secret_id"                   = ""
          + "vault_url"                   = ""
        }
    }

  # module.bigip[0].azurerm_network_interface.mgmt_nic[0] will be created
  + resource "azurerm_network_interface" "mgmt_nic" {
      + applied_dns_servers           = (known after apply)
      + dns_servers                   = (known after apply)
      + enable_accelerated_networking = false
      + enable_ip_forwarding          = false
      + id                            = (known after apply)
      + internal_dns_name_label       = (known after apply)
      + internal_domain_name_suffix   = (known after apply)
      + location                      = (known after apply)
      + mac_address                   = (known after apply)
      + name                          = (known after apply)
      + private_ip_address            = (known after apply)
      + private_ip_addresses          = (known after apply)
      + resource_group_name           = (known after apply)
      + tags                          = (known after apply)
      + virtual_machine_id            = (known after apply)

      + ip_configuration {
          + name                          = (known after apply)
          + primary                       = (known after apply)
          + private_ip_address            = (known after apply)
          + private_ip_address_allocation = "dynamic"
          + private_ip_address_version    = "IPv4"
          + public_ip_address_id          = (known after apply)
          + subnet_id                     = (known after apply)
        }
    }

  # module.bigip[0].azurerm_network_interface_security_group_association.mgmt_security[0] will be created
  + resource "azurerm_network_interface_security_group_association" "mgmt_security" {
      + id                        = (known after apply)
      + network_interface_id      = (known after apply)
      + network_security_group_id = (known after apply)
    }

  # module.bigip[0].azurerm_public_ip.mgmt_public_ip[0] will be created
  + resource "azurerm_public_ip" "mgmt_public_ip" {
      + allocation_method       = "Static"
      + domain_name_label       = (known after apply)
      + fqdn                    = (known after apply)
      + id                      = (known after apply)
      + idle_timeout_in_minutes = 4
      + ip_address              = (known after apply)
      + ip_version              = "IPv4"
      + location                = (known after apply)
      + name                    = (known after apply)
      + resource_group_name     = (known after apply)
      + sku                     = "Standard"
      + tags                    = (known after apply)
      + zones                   = [
          + "1",
        ]
    }

  # module.bigip[0].azurerm_user_assigned_identity.user_identity will be created
  + resource "azurerm_user_assigned_identity" "user_identity" {
      + client_id           = (known after apply)
      + id                  = (known after apply)
      + location            = (known after apply)
      + name                = (known after apply)
      + principal_id        = (known after apply)
      + resource_group_name = (known after apply)
    }

  # module.bigip[0].azurerm_virtual_machine.f5vm01 will be created
  + resource "azurerm_virtual_machine" "f5vm01" {
      + availability_set_id              = (known after apply)
      + delete_data_disks_on_termination = true
      + delete_os_disk_on_termination    = true
      + id                               = (known after apply)
      + license_type                     = (known after apply)
      + location                         = (known after apply)
      + name                             = (known after apply)
      + network_interface_ids            = (known after apply)
      + primary_network_interface_id     = (known after apply)
      + resource_group_name              = (known after apply)
      + tags                             = (known after apply)
      + vm_size                          = "Standard_DS3_v2"
      + zones                            = [
          + "1",
        ]

      + identity {
          + identity_ids = (known after apply)
          + principal_id = (known after apply)
          + type         = "UserAssigned"
        }

      + os_profile {
          + admin_password = (sensitive value)
          + admin_username = "bigipuser"
          + computer_name  = (known after apply)
          + custom_data    = (known after apply)
        }

      + os_profile_linux_config {
          + disable_password_authentication = true

          + ssh_keys {
              + key_data = <<-EOT
                    ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDkllReSMXoFn5LTGTnfKXiyDP4+rsvYNfK2bWkZ5G81NNRKhBljIw0UNGEqDa0omKrG0Iy3HqGVsy7g/uKZhtOPrXA74SePGhSaipWTPyftbqdF1PuvT4y3+D7V1zhiv29MejFY2+jd4LG1sGDb5jrkUZVlZUxUIhhJnZ97W43lQrpSk0381697Da55u9r+ko4XtN83k+ggmljCFV8/pCvih2UkpClzAjVW5s22oYFZ50OkDtgqOZp0PIasc35ZIAxugvaKIp9/1UJ5mAtx2wiWJ13NiNVtTvkqxZFtQGVmKHfQxuqYqURBbF5U1ENN3m/xEPLSLNAborozyoIo8kA8vxj2YRHD4XC/wv+Sl8DnDfXbH3y/HjiDMzXBB92HoQJ5wkswQkrUmjpoRUVV7MZr+R8ABnD6YhTlFUKNxXCmAacTQv02fA01gCVLFy6s2tkJ3/CnNOfoNmEOH0ICKiuNPyRH1lZ7jCATXrCCOXnSt6vyDWIkhm8HTZUySQAUf8= shitole@SEA-ML-00060393
                EOT
              + path     = "/home/bigipuser/.ssh/authorized_keys"
            }
        }

      + plan {
          + name      = "f5-bigip-virtual-edition-200m-best-hourly"
          + product   = "f5-big-ip-best"
          + publisher = "f5-networks"
        }

      + storage_data_disk {
          + caching                   = (known after apply)
          + create_option             = (known after apply)
          + disk_size_gb              = (known after apply)
          + lun                       = (known after apply)
          + managed_disk_id           = (known after apply)
          + managed_disk_type         = (known after apply)
          + name                      = (known after apply)
          + vhd_uri                   = (known after apply)
          + write_accelerator_enabled = (known after apply)
        }

      + storage_image_reference {
          + offer     = "f5-big-ip-best"
          + publisher = "f5-networks"
          + sku       = "f5-bigip-virtual-edition-200m-best-hourly"
          + version   = "14.1.206000"
        }

      + storage_os_disk {
          + caching                   = "ReadWrite"
          + create_option             = "FromImage"
          + disk_size_gb              = (known after apply)
          + managed_disk_id           = (known after apply)
          + managed_disk_type         = "Standard_LRS"
          + name                      = (known after apply)
          + os_type                   = (known after apply)
          + write_accelerator_enabled = false
        }
    }

  # module.bigip[0].azurerm_virtual_machine_extension.vmext will be created
  + resource "azurerm_virtual_machine_extension" "vmext" {
      + id                   = (known after apply)
      + name                 = (known after apply)
      + publisher            = "Microsoft.OSTCExtensions"
      + settings             = jsonencode(
            {
              + commandToExecute = "bash /var/lib/waagent/CustomData"
            }
        )
      + type                 = "CustomScriptForLinux"
      + type_handler_version = "1.2"
      + virtual_machine_id   = (known after apply)
    }

  # module.bigip[0].random_id.module_id will be created
  + resource "random_id" "module_id" {
      + b64_std     = (known after apply)
      + b64_url     = (known after apply)
      + byte_length = 2
      + dec         = (known after apply)
      + hex         = (known after apply)
      + id          = (known after apply)
    }

  # module.bigip[0].random_string.password will be created
  + resource "random_string" "password" {
      + id          = (known after apply)
      + length      = 16
      + lower       = true
      + min_lower   = 1
      + min_numeric = 1
      + min_special = 0
      + min_upper   = 1
      + number      = true
      + result      = (known after apply)
      + special     = false
      + upper       = true
    }

  # module.mgmt-network-security-group.data.azurerm_resource_group.nsg will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_resource_group" "nsg"  {
      + id       = (known after apply)
      + location = (known after apply)
      + name     = (known after apply)
      + tags     = (known after apply)

      + timeouts {
          + read = (known after apply)
        }
    }

  # module.mgmt-network-security-group.azurerm_network_security_group.nsg will be created
  + resource "azurerm_network_security_group" "nsg" {
      + id                  = (known after apply)
      + location            = (known after apply)
      + name                = (known after apply)
      + resource_group_name = (known after apply)
      + security_rule       = (known after apply)
      + tags                = {
          + "costcenter"  = "terraform"
          + "environment" = "dev"
        }
    }

  # module.network.data.azurerm_resource_group.vnet will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_resource_group" "vnet"  {
      + id       = (known after apply)
      + location = (known after apply)
      + name     = (known after apply)
      + tags     = (known after apply)

      + timeouts {
          + read = (known after apply)
        }
    }

  # module.network.azurerm_subnet.subnet[0] will be created
  + resource "azurerm_subnet" "subnet" {
      + address_prefix                                 = (known after apply)
      + address_prefixes                               = [
          + "10.2.1.0/24",
        ]
      + enforce_private_link_endpoint_network_policies = false
      + enforce_private_link_service_network_policies  = false
      + id                                             = (known after apply)
      + name                                           = "mgmt-subnet"
      + resource_group_name                            = (known after apply)
      + virtual_network_name                           = (known after apply)
    }

  # module.network.azurerm_virtual_network.vnet will be created
  + resource "azurerm_virtual_network" "vnet" {
      + address_space         = [
          + "10.2.0.0/16",
        ]
      + dns_servers           = []
      + guid                  = (known after apply)
      + id                    = (known after apply)
      + location              = (known after apply)
      + name                  = (known after apply)
      + resource_group_name   = (known after apply)
      + subnet                = (known after apply)
      + tags                  = {
          + "costcenter"  = "it"
          + "environment" = "dev"
        }
      + vm_protection_enabled = false
    }

Plan: 17 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + bigip_password     = [
      + (known after apply),
    ]
  + bigip_username     = [
      + "bigipuser",
    ]
  + mgmtPort           = [
      + "8443",
    ]
  + mgmtPublicDNS      = [
      + (known after apply),
    ]
  + mgmtPublicIP       = [
      + (known after apply),
    ]
  + mgmtPublicURL      = [
      + (known after apply),
    ]
  + private_addresses  = [
      + [],
    ]
  + public_addresses   = [
      + [],
    ]
  + resourcegroup_name = (known after apply)

Warning: Version constraints inside provider configuration blocks are deprecated

  on main.tf line 2, in provider "azurerm":
   2:   version = "~>2.0"

Terraform 0.13 and earlier allowed provider version constraints inside the
provider configuration block, but that is now deprecated and will be removed
in a future version of Terraform. To silence this warning, move the provider
version constraint into the required_providers block.


------------------------------------------------------------------------

Note: You didn't specify an "-out" parameter to save this plan, so Terraform
can't guarantee that exactly these actions will be performed if
"terraform apply" is subsequently run.

SEA-ML-00060393:bigip_azure_1nic_deploy shitole$ terraform apply -auto-approve
module.bigip[0].random_id.module_id: Creating...
random_id.id: Creating...
module.bigip[0].random_string.password: Creating...
random_id.id: Creation complete after 0s [id=ebY]
module.bigip[0].random_id.module_id: Creation complete after 0s [id=wvI]
module.bigip[0].random_string.password: Creation complete after 1s [id=1VU3Q5WyzKZkAz8Z]
module.bigip[0].data.template_file.init_file[0]: Reading...
module.bigip[0].data.template_file.init_file[0]: Read complete after 0s [id=3a864e72b7036173cb2bf444674fa31b192e248857ce1f59abc66598d42690e5]
azurerm_resource_group.rg: Creating...
azurerm_resource_group.rg: Creation complete after 1s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6]
module.network.data.azurerm_resource_group.vnet: Reading...
module.mgmt-network-security-group.data.azurerm_resource_group.nsg: Reading...
module.bigip[0].data.azurerm_resource_group.bigiprg: Reading...
module.network.data.azurerm_resource_group.vnet: Read complete after 0s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6]
module.bigip[0].data.azurerm_resource_group.bigiprg: Read complete after 0s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6]
module.mgmt-network-security-group.data.azurerm_resource_group.nsg: Read complete after 0s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6]
module.bigip[0].azurerm_user_assigned_identity.user_identity: Creating...
module.mgmt-network-security-group.azurerm_network_security_group.nsg: Creating...
module.network.azurerm_virtual_network.vnet: Creating...
module.bigip[0].azurerm_user_assigned_identity.user_identity: Creation complete after 5s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.ManagedIdentity/userAssignedIdentities/tf-azure-bigip-1nic-c2f2-ident]
module.network.azurerm_virtual_network.vnet: Creation complete after 7s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/virtualNetworks/tf-azure-bigip-vnet-79b6]
module.network.azurerm_subnet.subnet[0]: Creating...
module.mgmt-network-security-group.azurerm_network_security_group.nsg: Creation complete after 8s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkSecurityGroups/tf-azure-bigip-mgmt-nsg-79b6]
azurerm_network_security_rule.mgmt_allow_http: Creating...
azurerm_network_security_rule.mgmt_allow_https: Creating...
azurerm_network_security_rule.mgmt_allow_ssh: Creating...
module.network.azurerm_subnet.subnet[0]: Creation complete after 4s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/virtualNetworks/tf-azure-bigip-vnet-79b6/subnets/mgmt-subnet]
data.azurerm_subnet.mgmt: Reading...
data.azurerm_subnet.mgmt: Read complete after 1s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/virtualNetworks/tf-azure-bigip-vnet-79b6/subnets/mgmt-subnet]
module.bigip[0].azurerm_public_ip.mgmt_public_ip[0]: Creating...
azurerm_network_security_rule.mgmt_allow_https: Still creating... [10s elapsed]
azurerm_network_security_rule.mgmt_allow_ssh: Still creating... [10s elapsed]
azurerm_network_security_rule.mgmt_allow_http: Still creating... [10s elapsed]
module.bigip[0].azurerm_public_ip.mgmt_public_ip[0]: Creation complete after 7s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/publicIPAddresses/tf-azure-bigip-1nic-c2f2-pip-mgmt-0]
module.bigip[0].azurerm_network_interface.mgmt_nic[0]: Creating...
azurerm_network_security_rule.mgmt_allow_ssh: Creation complete after 12s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkSecurityGroups/tf-azure-bigip-mgmt-nsg-79b6/securityRules/Allow_ssh]
module.bigip[0].azurerm_network_interface.mgmt_nic[0]: Creation complete after 4s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkInterfaces/tf-azure-bigip-1nic-c2f2-mgmt-nic-0]
module.bigip[0].azurerm_network_interface_security_group_association.mgmt_security[0]: Creating...
azurerm_network_security_rule.mgmt_allow_https: Still creating... [20s elapsed]
azurerm_network_security_rule.mgmt_allow_http: Still creating... [20s elapsed]
azurerm_network_security_rule.mgmt_allow_http: Creation complete after 23s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkSecurityGroups/tf-azure-bigip-mgmt-nsg-79b6/securityRules/Allow_Http]
module.bigip[0].azurerm_network_interface_security_group_association.mgmt_security[0]: Still creating... [10s elapsed]
azurerm_network_security_rule.mgmt_allow_https: Still creating... [30s elapsed]
azurerm_network_security_rule.mgmt_allow_https: Creation complete after 34s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkSecurityGroups/tf-azure-bigip-mgmt-nsg-79b6/securityRules/Allow_Https]
module.bigip[0].azurerm_network_interface_security_group_association.mgmt_security[0]: Still creating... [20s elapsed]
module.bigip[0].azurerm_network_interface_security_group_association.mgmt_security[0]: Creation complete after 21s [id=/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkInterfaces/tf-azure-bigip-1nic-c2f2-mgmt-nic-0|/subscriptions/187fa2f3-5d57-4e6a-9b1b-f92ba7adbf42/resourceGroups/tf-azure-bigip-rg-79b6/providers/Microsoft.Network/networkSecurityGroups/tf-azure-bigip-mgmt-nsg-79b6]
module.bigip[0].azurerm_virtual_machine.f5vm01: Creating...

Warning: Version constraints inside provider configuration blocks are deprecated

  on main.tf line 2, in provider "azurerm":
   2:   version = "~>2.0"

Terraform 0.13 and earlier allowed provider version constraints inside the
provider configuration block, but that is now deprecated and will be removed
in a future version of Terraform. To silence this warning, move the provider
version constraint into the required_providers block.


Error: compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: Code="PlatformImageNotFound" Message="The platform image 'f5-networks:f5-big-ip-best:f5-bigip-virtual-edition-200m-best-hourly:14.1.206000' is not available. Verify that all fields in the storage profile are correct. For more details about storage profile information, please refer to https://aka.ms/storageprofile" Target="imageReference"

  on ../../main.tf line 464, in resource "azurerm_virtual_machine" "f5vm01":
 464: resource "azurerm_virtual_machine" "f5vm01" {


SEA-ML-00060393:bigip_azure_1nic_deploy shitole$ 

```

**Comment by nmenant** [comment]
Hi,

This is because by default we use BIG-IP VE image 14.1 b206 and it has been deprecated. We have already updated the code to deploy the latest version available in the future. Tracked internally with INFRAANO-196

In the meantime, you can edit this section of the module to workaround the issue: 
https://github.com/f5devcentral/terraform-azure-bigip-module/blob/master/variables.tf

``` 
variable f5_version {
  type    = string
  default = "14.1.206000"
}
```

**Comment by RavinderReddyF5** [comment]
@scshitole fixed in v0.9.4, please verify
